frontend/src/screens/SingleVideo/index.js

```diff
@@ -15,18 +17,15 @@ const SingleVideo = () => {
     const videoId = query.get('v');
     const { state, dispatch } = useCustomContext();
     const { singleVideo } = state;
-    const [show, setShow] = useState(false);
     const [showModal, setShowModal] = useState(false);
 
     const getVideoDetails = async () => {
         try {
             dispatch({ type: SINGLE_VIDEO_REQUEST })
-            const response = await axios.get(`https://youtube.googleapis.com/youtube/v3/videos?part=snippet%2Cstatistics&id=${videoId}&key=${process.env.REACT_APP_YOUTUBE_API_KEY}`);
-
+            const response = await axios.get(`${process.env.REACT_APP_SERVER_URL}/api/videos/video/${videoId}`, { headers: { 'x-auth-token': localStorage.getItem('token') } });
             if (response.status === 200) {
-                const { data: { items } } = response;
-
-                dispatch({ type: SINGLE_VIDEO_SUCCESS, payload: items[0] })
+                const { data } = response;
+                dispatch({ type: SINGLE_VIDEO_SUCCESS, payload: data })
             }
         } catch (err) {
             dispatch({ type: SINGLE_VIDEO_FAILURE, payload: 'Something went wrong' })
```

Comment:
Okay!
So here since my Youtube videos are coming from Youtube-API itself and in video.controller.js file I have made wrapper and 
for error handling of this didn't any custom error message because if error occurs it would come from Youtube-API itself and
in that case frontend would break abruptly therefore for videos management I have handled its error in frontend itself
and therefore not used toast.error here